Core/include/Acts/TrackFinding/MeasurementSelector.hpp

```diff
@@ -36,8 +36,10 @@ namespace Acts {
 struct MeasurementSelectorCuts {
   /// bins in |eta| to specify variable selections
   std::vector<double> etaBins{};
-  /// Maximum local chi2 contribution.
+  /// Maximum local chi2 contribution to classify as outlier.
   std::vector<double> chi2CutOff{15};
```

Comment:
In the most likely case that the vector only contains one element this does not matter. Otherwise double precision for the chi2 cut feels over precise and wastes cache.